locked-asset/locked-token-wrapper/src/lib.rs

```diff
@@ -71,17 +49,11 @@ pub trait LockedTokenWrapper:
         let payment = self.call_value().single_esdt();
         let original_locked_tokens = self.unwrap_locked_token(payment);
 
-        if original_locked_tokens.token_identifier == self.locked_token().get_token_id() {
+        if original_locked_tokens.token_identifier == self.get_locked_token_id() {
```

Comment:
when is locked_token_id set ?

Reply:
It is no longer saved. We read it directly from the energy factory storage.